src/Mvc/test/WebSites/BasicWebSite/Views/Components/Index.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@using BasicWebSite.RazorComponents;
+<h1>Razor components</h1>
+<div id="Greetings">
+    @(await Html.RenderComponentAsync<Greetings>())
```

Comment:
@NTaylorMullen / @ajaybhargavb - have we considered a change to make this work with an implicit expression?

Reply:
Yes, however we can't make it work in the general case because we can't easily differentiate: `@await Html.RenderComponentAsync<Greetings>()` from `@await NameTask<strong>(First Name)</strong>`.

Reply:
that seems like a bad choice on our part :( OK

Reply:
@rynowak I would expect in the future that this experience is more integrated. I imagine support for this within razor itself (when it detects a component inside a view it knows what to do and offers appropriate intellisense) is that crazy?

Reply:
it's planned that you can **just** use a component from a view (eventually)